hackathon_site/dashboard/frontend/src/slices/event/teamSlice.ts

```diff
@@ -77,6 +161,47 @@ const teamSlice = createSlice({
         builder.addCase(getCurrentTeam.rejected, (state, { payload }) => {
             state.error = payload?.message || "Something went wrong";
         });
+
+        builder.addCase(leaveTeam.pending, (state) => {
+            state.isLeaveTeamLoading = true;
+            state.error = null;
+        });
+
+        builder.addCase(
+            leaveTeam.fulfilled,
+            (state, { payload }: PayloadAction<Team>) => {
+                if (payload) {
```

Comment:
don't need this condition here because payload will always be a non-null value

Reply:
you are right yuying